### PR TITLE
Updated hydrogen

### PR DESCRIPTION
**Pull request by timowestnosto**

### CODE REVIEW

File: src/hooks/useClientScriptLoad.ts [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+	
+			if (!isNostoLoaded() && !shopifyMarkets && loadScript) {
+				const script = document.createElement("script")
+				script.type = "text/javascript"
+				script.src = "//" + (host || "connect.nosto.com") + "/include/" + account
+				script.async = true
+				script.setAttribute("nosto-client-script", "")
+
+				script.onload = () => {
+				if (typeof "nostoReactTest" !== "undefined") {
```
Comment (timowestnosto) [inline src/hooks/useClientScriptLoad.ts:29]: I would replace this line with the equivalent line in master
typeof "nostoReactTest" is "string"

File: src/hooks/useClientScriptLoad.ts [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+							`/script/shopify/market/nosto.js?merchant=${account}&market=${
+							shopifyMarkets.marketId || ""
+							}&locale=${shopifyMarkets?.language?.toLowerCase() || ""}`
+					script.async = true
+					script.setAttribute("nosto-client-script", "")
+					script.setAttribute("nosto-language", shopifyMarkets?.language || "")
+					script.setAttribute("nosto-market-id", String(shopifyMarkets?.marketId))
+	
+					script.onload = () => {
+						if (typeof "nostoReactTest" !== "undefined") {
```
Comment (timowestnosto) [inline src/hooks/useClientScriptLoad.ts:70]: same as above

File: src/hooks/useClientScriptLoad.ts (hunk 1 above)
Comment (nosto-mikpou) [reply src/hooks/useClientScriptLoad.ts:29]: The `typeof "nostoReactTest" in window` seems sufficient for me

File: src/hooks/useClientScriptLoad.ts [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+
+		if (!isNostoLoaded() && !shopifyMarkets && loadScript) {
+			const script = document.createElement("script")
+			script.type = "text/javascript"
+			script.src = "//" + (host || "connect.nosto.com") + "/include/" + account
+			script.async = true
+			script.setAttribute("nosto-client-script", "")
+
+			script.onload = () => {
+				if (typeof "nostoReactTest" in window) {
```
Comment (timowestnosto) [inline src/hooks/useClientScriptLoad.ts:29]: this is being duplicated in this file now
also typeof & in mixed like there doesn't really work
fix the logic and extract to function?

**Pull request closed without merging**